src/kata2.booleanToWord.js

```diff
@@ -1,5 +1,12 @@
 const booleanToWord = (boolean) => {
-
-}
+    if(boolean === Boolean(true))
+    {
+        return true;
+    }
+    else if(boolean === Boolean(false))
+    {
+        return false;
+    }
```

Comment:
`else if` should never exist without an `else` statement. Note also that your test isn't testing for the correct return values, so this code is actually incorrect.